eng/tools/spec-gen-sdk-runner/src/commands.ts

```diff
@@ -497,15 +506,20 @@ function generateArtifact(
       fs.mkdirSync(specGenSdkArtifactAbsoluteFolder, { recursive: true });
     }
     // Write artifact
-    artifactInfo.language = commandInput.sdkLanguage;
-    artifactInfo.labelAction = shouldLabelBreakingChange;
+    const artifactInfo: SpecGenSdkArtifactInfo = {
+      language: commandInput.sdkLanguage,
+      result,
+      labelAction: hasBreakingChange,
+      isSpecGenSdkCheckRequired:
```

Comment:
Making required as long as a spec PR contains management plane specs.

Reply:
This is fine for now but if the logic gets more involved we should factor it out into its own method.